### PR TITLE
Feat/docker container on main builds

### PR DESCRIPTION
**Pull request by gardleopard**
This makes it easy to build docker containers locally from branches and publishes a development version of unleash to dockerhub.

### CODE REVIEW

**Review by chriswk — APPROVED** [review]
LGTM

**Review by chriswk — APPROVED** [review]
This will make the tag be `main-edge-14-alpine` and `main-edge-16-alpine`
I'm fine with that :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -103,10 +103,37 @@ You'll need:
...
    yarn start:dev
    ```
 
+5. **Log into the admin UI**. Use a browser and navigate to `localhost:4242`. Log in using:
+   - username: `admin`
+   - password: `unleash4all`
+
+### Steps running locally with docker
+
+1. Build local docker image by running `docker build . -t unleash:local`
```
Comment (thomasheartman) [inline CONTRIBUTING.md:117]:
```suggestion
1. Build a local docker image by running `docker build . -t unleash:local`
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -103,10 +103,37 @@ You'll need:
...
+2. Create a network by running `docker network create unleash`
+3. Start a postgres database:
```
Comment (thomasheartman) [inline CONTRIBUTING.md:119]:
```suggestion
3. Start a Postgres database. Make sure to use the network you created in step 2.
```

**Review by thomasheartman — CHANGES_REQUESTED** [review]
Nice one! 🙌🏼  Added a couple small suggestions, but this looks great in general. Thanks for adding the steps! 😄

**Review by thomasheartman — APPROVED** [review]
🚂

**Pull request merged**